json-path/src/test/java/com/jayway/jsonpath/JsonOrgJsonProviderTest.java

```diff
@@ -76,4 +78,23 @@ public void read_book_length() {
         assertThat(result).isEqualTo(4);
     }
 
+    @Test
+    public void Issue497() {
```

Comment:
Please change method names to something more descriptive. I don't want to have to go to github to find out what this issue is.